### PR TITLE
Move serializers setup to jest config to be possible to override them

### PR DESCRIPTION
**Pull request by yurii-sorokin**
#125

### CODE REVIEW

File: jest-preset.json [hunk 1]
```diff
@@ -24,5 +24,9 @@
   },
   "transformIgnorePatterns": [
     "node_modules/(?!@ngrx)"
+  ],
+  "snapshotSerializers": [
+    "<rootDir>/node_modules/jest-preset-angular/AngularSnapshotSerializer.js",
+    "<rootDir>/node_modules/jest-preset-angular/HTMLCommentSerializer.js"
```
Comment (thymikee) [inline jest-preset.json]: Shouldn't the order be different? Have you tested if that changes anything?

Comment (yurii-sorokin) [reply jest-preset.json]: Just tried both and seems there is no any difference. Will look how jest applies this config.

Comment (yurii-sorokin) [reply jest-preset.json]: I bet I've seen something about the order in jest docs, but can't find it now. Anyway, jest reverses the list:
```
config.snapshotSerializers
    .concat()
    .reverse()
    .forEach(path => {
      addSerializer(localRequire(path));
    });
```
So it's same as before.

**Comment by AhnpGit** [comment]
nice 👍

**Comment by thymikee** [comment]
published v5.1.0 with this change

**Comment by ewantavenerdemo** [comment]
Does this mean I can now change the config for 'pretty-format'?

**Comment by yurii-sorokin** [comment]
You can try [this serializer](https://github.com/yurii-sorokin/ngx-jest-snapshot-serializer) that uses the default serializers (built-in plugins for `pretty-format`). Basically, it transforms component fixture to react test object (mostly, as it does not convert dom nodes needed when using `[innerHTML]`). Which `jest-snapshot` understand well.

Or you can write your own serializer, for example, that uses `require('util').inspect` to serialize fixture or anything you like.

**Pull request merged**